src/NuGetGallery.Core/GitHub/GitHubUsageConfiguration.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGetGallery.GitHub
+{
+    public class GitHubUsageConfiguration : IGitHubUsageConfiguration
+    {
+        public const int ReposPerPackage = 10;
+        private readonly Dictionary<string, NuGetPackageGitHubInformation> _nuGetPackagesGitHubDependencies;
+
+        public GitHubUsageConfiguration(IReadOnlyList<RepositoryInformation> repositories)
+        {
+            if (repositories == null)
+            {
+                throw new ArgumentNullException(nameof(repositories));
+            }
+
+            _nuGetPackagesGitHubDependencies = repositories.Any()
+                ? GetNuGetPackagesDependents(repositories)
```

Comment:
`GetNuGetPackagesDependents` does the right thing even if `repositories` is empty right? If so, I would simplify this code to just call `GetNuGetPackagesDependents` regardless